include/config.h

```diff
@@ -6,4 +6,5 @@
 
 #pragma once
 
+#include <autoconf.h>
```

Comment:
This is already included by sel4/config.h.  Why include it again here?

Reply:
I was not sure what policy we have, "Include what you need" or "trust others". but we can remove it, if there is a clear agreement that `sel4/config.h` is supposed to provide this (and we we want to drop more autoconf legacy). I will remove it then.